mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -119,7 +119,7 @@ case class ScalaPresentationCompiler(
       val sourceFile = CompilerInterfaces.toSource(params.uri, params.text)
       driver.run(uri, sourceFile)
 
-      given ctx as Context = driver.currentCtx
+      val ctx = driver.currentCtx
```

Comment:
🤔 This isn't 100% clear to me. How come this doesn't need to be a `given` anymore?

Reply:
This is a workaround not to use givens, because they changed the syntax and I would have duplicate the whole file. By doing it all explicitely it seems to work fine.